contentcuration/contentcuration/frontend/channelList/views/Channel/CatalogFAQ.vue

```diff
@@ -349,7 +349,7 @@
         'Here you can learn more about the educational resources publicly available for use in Kolibri, which are organized into "channels". Use the filters to browse channels by keyword, language, or formats of the materials inside.',
       channelLink: 'What is a channel?',
       descriptionP2:
-        'Click on a channel to get a preview of what subjects and topics it covers, learn more about its creator, see how many resources the channel contains, and learn how to import it into Kolibri. You can also find coach-specific content (lesson plans, teacher professional guides, and other supplementary facilitation material), assessments and exercises, and captions for accessibility.',
+        'Click on a channel to get a preview of what subjects and folders it covers, learn more about its creator, see how many resources the channel contains, and learn how to import it into Kolibri. You can also find coach-specific content (lesson plans, teacher professional guides, and other supplementary facilitation material), assessments and exercises, and captions for accessibility.',
```

Comment:
I'm not sure if here it makes more sense to actually leave `topics`, as _...what subjects and folders it covers_ sounds awkward to me... 🤔 

Thoughts, @navakk9, @lauradanforth, @carinediaz & @laurenlichtman?

Reply:
I have changed it back to topics since it makes more sense to me as well.